grpc/src/main/java/com/replica/CustomStreetLeg.java

```diff
@@ -18,7 +18,6 @@ public CustomStreetLeg(Trip.WalkLeg leg, List<String> stableEdgeIds, String trav
         super(leg.departureLocation, leg.getDepartureTime(), leg.geometry,
                 leg.distance, leg.instructions, leg.details, leg.getArrivalTime());
         this.stableEdgeIds = stableEdgeIds;
-        this.details.clear();
```

Comment:
I traced through the creation of the proto object we create as a final product for street-based legs in transit responses, and this field is never used. Given the new empty walking legs are setting `details` as `null`, it felt clunky to add an `if` handling this case to prevent a NPE vs. just getting rid of this line, which seems unnecessary